.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,9 +3,13 @@
 
 ## Checklist
 
+- [ ] The PR has a meaningful title. It will be used to auto generate the
+      changelog.
+      The PR has a meaningful description that sums up the change. It will be
```

Comment:
```suggestion
 - []  The PR has a meaningful description that sums up the change. It will be
```
I would add `[ ]` for the second sentence as well.

Reply:
@glrf suggested combining it into the same checkbox. I don't have any hard feelings for either of the solutions. But your suggestion basically reverts his.

Reply:
I don't have strong feelings. It's just another click we'll have to do every time :sweat_smile:

Reply:
ahaha fine by me.